### PR TITLE
Doc gen

### PR DESCRIPTION
**Pull request by slemasne**

### CODE REVIEW

File: docgen/parser.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+def build_mustache_data(files_in_scope, project_name):
+
+    """
+
+    :param files_in_scope: These are the files and dirs in scope
+    :param project_name: The main project name
+    :return: A list of dicts which can be passed to the mustache template
+    """
+    
+    file_date_for_template = []
```
Comment (DavidWassellFinbourne) [inline docgen/parser.py:63]: Do you want this to say data or did you mean date?

**Review by DavidWassellFinbourne — COMMENTED** [review]
All good other than the 1 comment made. If this is as it should be, then can approve.

File: docgen/parser.py (hunk 1 above)
Comment (slemasne) [reply docgen/parser.py]: good catch, updated

**Review by DavidWassellFinbourne — APPROVED** [review]
All looks good.

**Pull request merged**